Libplanet/Store/NamespaceNotFoundException.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Libplanet.Store
+{
+    /// <summary>
+    /// The exception that is thrown when the namespace of
+    /// <see cref="Libplanet.Store"/> does not exist.
+    /// </summary>
+    [Serializable]
+    public sealed class NamespaceNotFoundException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the
+        /// <see cref="NamespaceNotFoundException"/> class.
+        /// </summary>
+        /// <param name="message">The message that describes the error.</param>
+        public NamespaceNotFoundException(string message)
```

Comment:
It would be good if this exception has a property to refer to the namespace tried to find.